cmd/restic/cmd_ls.go

```diff
@@ -59,19 +71,55 @@ func runLs(opts LsOptions, gopts GlobalOptions, args []string) error {
 		return err
 	}
 
+	// extract any specific directories to walk
+	var dirs []string
+	if len(args) > 1 {
+		dirs = args[1:]
+	}
+
 	ctx, cancel := context.WithCancel(gopts.ctx)
 	defer cancel()
-	for sn := range FindFilteredSnapshots(ctx, repo, opts.Host, opts.Tags, opts.Paths, args) {
-		Verbosef("snapshot %s of %v at %s):\n", sn.ID().Str(), sn.Paths, sn.Time)
+	for sn := range FindFilteredSnapshots(ctx, repo, opts.Host, opts.Tags, opts.Paths, args[:1]) {
+		dirsToPrint := opts.Paths
```

Comment:
Hm, I think the message should always print the source paths the snapshot was taken of. This is something completely different from the path within a snapshot. For example, consider a snapshot created with the following command:
```
$ cd /home/user
$ restic backup work
```

Then there will be a single path in the snapshot called `/home/user/work`, but the top-level directory in the snapshot will be `/work`. That's confusing for users, I think.

Reply:
Alright -- I can understand that. Reverting this change!